rootfs/deploy.py

```diff
@@ -36,6 +36,39 @@ def log(msg):
         print(msg)
 
 
+def get_registry_name():
+    hostname = os.getenv('DEIS_REGISTRY_HOSTNAME', "").replace("https://", "")
```

Comment:
why would we want to strip the protocol only if it's https? Should we also strip on http protocols too? What's the intent here?